Sonoff-Tasmota Driver/xsns_91_I2cRotary.ino

```diff
@@ -1,3 +1,6 @@
+#define XSNS_92             92
+
+
 #ifdef USE_I2C
 #ifdef USE_I2C_ROTARY
```

Comment:
activate the Driver within User_Config_Override.h with

#define USE_I2C_ROTARY